Kuzzle/Utils/QueryUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace KuzzleSdk.Utils {
```

Comment:
After reading @benoitvidis 's review I actually agree: this needs to be isolated in its own namespace, such as `KuzzleSdk.Api.QueryOptions` for instance

Reply:
I've changed how query options are handled.
Now QueryAsync takes an QueryOptions as optional parameter.